Src/Newtonsoft.Json.UnityConverters.Tests/ConvertingUnityTypes/TypeTester.cs

```diff
@@ -1,25 +1,63 @@
-﻿using Newtonsoft.Json.Linq;
+﻿using System.Diagnostics.CodeAnalysis;
+using System.IO;
+using System.Text;
 using NUnit.Framework;
 
 namespace Newtonsoft.Json.UnityConverters.Tests.ConvertingUnityTypes
 {
     public abstract class TypeTesterBase
     {
-        protected virtual void ConfigureSettings(JsonSerializerSettings settings)
+        protected readonly JsonSerializer _serializer;
+
+        static TypeTesterBase()
+        {
+            UnityConverterInitializer.ShouldAddConvertsToDefaultSettings = false;
+            UnityConverterInitializer.DefaultUnityConvertersSettings.Formatting = Formatting.None;
+        }
+
+        protected TypeTesterBase()
+        {
+            _serializer = JsonSerializer.Create(UnityConverterInitializer.DefaultUnityConvertersSettings);
+            ConfigureSerializer(_serializer);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove this call from a constructor to the overridable 'ConfigureSerializer' method.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5420094)